sites/fast-website/webpack.common.js

```diff
@@ -59,19 +58,18 @@ module.exports = {
                     {
                         loader: "svg-inline-loader",
                         options: {
-                            removeSVGTagAttrs: false
-                        }
-                    }
-                ]
-            }
+                            removeSVGTagAttrs: false,
+                        },
+                    },
+                ],
+            },
         ],
     },
     plugins: [
         new CleanWebpackPlugin(),
         new HtmlWebpackPlugin({
-            title: "FAST website",
+            title: "FAST",
```

Comment:
Could you inform more in the title for SEO? "FAST - An adaptive UI design system" for example on the home page or whatever our tagline may be?